includes/wc-core-functions.php

```diff
@@ -43,6 +43,9 @@
 /**
  * Short Description (excerpt).
  */
+if ( version_compare( $wp_version, '5.0', '>=' ) ) {
```

Comment:
$wp_version might not be defined here? Why not use `function_exists`?

Reply:
Functions don't need to exist when they are added to the hook/filter so I have tended to avoid using that for a check. I'll update the PR sometime tomorrow.

Reply:
Updated to function_exists.